index.html

```diff
@@ -68,6 +68,12 @@ <h2>
       <div class="alert alert-danger" ng-show="securityError" ng-cloak>
         <strong>Secure connection error</strong> Unable to connect to unencrypted relay when you are connecting to Glowing Bear over HTTPS. Please use an encrypted relay or load the page without using HTTPS.
       </div>
+      <div class="alert alert-danger" ng-show="oldWeechatError" ng-cloak>
+        <strong>Weechat version error</strong> Weechat connected but did not respond to a handshake. This could mean weechat < version 2.9. Verify your weechat version and check "Allow Plaintext Authentication" if it's < 2.9.
+      </div>
+      <div class="alert alert-danger" ng-show="hashAlgorithemDisagree" ng-cloak>
+        <strong>Hash algorithem error</strong> Weechat and glowing bear did not agree on a hashing algorithem, please do /set relay.network.password_hash_algo "pbkdf2+sha512" in weechat.
```

Comment:
The word is "algorithm" without an "e" ;)

Reply:
Agree to dissagree, That's just too many consonants in a row... ;) (fixed all occurenced)